.eslintrc.js

```diff
@@ -190,9 +190,11 @@ module.exports = {
     'unicorn/no-process-exit': 'off',
     'unicorn/no-useless-undefined': 'off',
     'unicorn/numeric-separators-style': 'off',
+    'unicorn/prefer-at': 'off',
```

Comment:
@sroy3, @julieg18 I actually like this one but it will require some code updates. WDYT?